### DISCUSSION TITLE
Using other repositories like GitLab and Bitbucket

### DISCUSSION THREAD

**Discussion by someotherguyontheweb** [cc-tweaked/CC-Tweaked#960] [Q&A] [answered]
Sometimes Friends ask me this question, I know how to put and get code using pastebin, but I don't know how it actually works, and when they ask me if there's a way to receive other repositories, I genuinely don't have the answer to it, so maybe the community can help me figure this out.

How does one get code from a website like GitLab or Bitbucket?

They both have a SHA code and branches set up very similar as right here on GitHub, however when attempting to pull for example: 
`> pastebin get 1234abcd main/Example.lua`
the return that you get on the terminal is: 
`Connecting to pastebin.com... Failed.`
This is presumably the default website.

The computercraft.cfg has unrestricted access to the internet, so is it even possible to get code from public repositories outside of pastebin.com? Any reply is much appreciated!

**Comment by Wojbie** [comment, marked as answer]
For downloading arbitrary files you are looking for `wget` program.

You just give it url to file and it downloads it.

Edit: Oh you might be asking to add support to git-like pages to pastebin? Normally this kind of additions were left to community.. I do believe there are programs for CC that emulate `pastebin` program for other websites. `gitget` comes to mind (Sorry no link). But for any file that you can get direct download url `wget` will download it fine.

**Reply by someotherguyontheweb** [reply]
so in terms I'd use wget like `wget <url> Example.lua`  sweet, that was very helpful, thank you!
